.circleci/config.yml

```diff
@@ -58,21 +58,20 @@ workflows:
             - hmpps-approved-premises-api-preprod
           requires:
             - request-preprod-approval
-
-#      - request-prod-approval:
-#          type: approval
-#          requires:
-#            - deploy_preprod
-#      - hmpps/deploy_env:
-#          name: deploy_prod
-#          env: "prod"
-#          slack_notification: true
-#          slack_channel_name: << pipeline.parameters.releases-slack-channel >>
-#          context:
-#            - hmpps-common-vars
-#            - approved-premises-api-prod
-#          requires:
-#            - request-prod-approval
+      - request-prod-approval:
+          type: approval
+          requires:
+            - deploy_preprod
+      - hmpps/deploy_env:
+          name: deploy_prod
+          env: "prod"
+          slack_notification: true
+          slack_channel_name: << pipeline.parameters.releases-slack-channel >>
+          context:
+            - hmpps-common-vars
+            - approved-premises-api-prod
+          requires:
+            - request-prod-approval
```

Comment:
perhaps we want to see failures in Slack? We could copy the config from MRD:

https://github.com/ministryofjustice/make-recall-decision-api/blob/41ebdcc8d4c618caa6e99af65821f8ec0ac23e95/.circleci/config.yml#L18-L46

Reply:
I hope [this standard config](https://github.com/ministryofjustice/hmpps-template-typescript/blob/main/.circleci/config.yml#L177) does send failures to Slack. We could definitely copy if that isn't what happens or want to add more custom messages 👍